### DISCUSSION TITLE
Can't access Media Library Pro repo

### DISCUSSION THREAD

**Discussion by GreenImp** [spatie/laravel-medialibrary#2747] [Q&A] [answered]
Hi,
We had a license for Media Library Pro, that has recently expired. I understand that we are not now entitled access to newer versions of Media Library Pro, as the license "Includes a year of updates".
However, it appears that we can also no longer access older versions of the library, that our license allowed.

I've just tried running a composer install, and get the following:

>  Failed to download spatie/laravel-medialibrary-pro from dist: Invalid credentials for 'https://satis.spatie.be/dist/spatie/laravel-medialibrary-pro/spatie-laravel-medialibrary-pro-07886cc56faccc1eef06466f8d67a1f3aae5e06d-zip-c2d29a.zip', aborting.
    Now trying to download from source

It is trying to install v1.17.0, which we were able to install before the license expired, so I thought we'd still be able to after it had expired, and that we'd just be locked in at that version.

Is this not the case? Can we no longer use the pro library (Even older versions) unless we pay for a new license each year?

**Comment by freekmurze** [comment, marked as answer]
Hi, we've documented what you can do when your license expires on this page: https://spatie.be/docs/laravel-medialibrary/v9/handling-uploads-with-media-library-pro/installation#what-happens-when-your-license-expires

**Reply by GreenImp** [reply]
Thanks @freekmurze I hadn't seen that! That makes it absolutely clear :smile:
Unfortunately, I don't control the email account used, so didn't see the reminders. I'll get on to management to get it sorted!
